crdstore/crdstore.go

```diff
@@ -95,6 +95,7 @@ func (cs *CredentialsStore) updateCache(accessKey, backend, key string, csd *Cre
 }
 
 func (cs *CredentialsStore) tryLock() bool {
+	// #nosec
 	return atomic.CompareAndSwapInt32((*int32)(unsafe.Pointer(&cs.lock)), 0, 1)
```

Comment:
Please add comment why it's done that way

Reply:
We'll get rid of this construct in future version. I promise :)